### PR TITLE
Addfilenametype

### PR DESCRIPTION
**Pull request by m-i-sadowski**
Sugaring of output and input filenames... mostly for convenience of front-end

### CODE REVIEW

File: antha/anthalib/wtype/filename.go [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package wtype
+
+type InputFilename string
+type OutputFilename string
```
Comment (gogogadgetbioprocess) [inline antha/anthalib/wtype/filename.go]: Could the distinction of these into two types become cumbersome?

e.g. in situations where an outputfilename is an inputfilename to a subsequent step?

Comment (m-i-sadowski) [inline antha/anthalib/wtype/filename.go]: Yeah, actually i'll just revert to a single type

**Comment by ddn0** [comment]
To allow for expanding the data representation, can we do something like:

```go
package wtype

type File struct {
  Name string
  Data []byte
  ...
}
```

instead of `type Filename string` ?

RFC: changing the name to File because this will be the type we use to transfer data blobs between elements and to the database eventually.

**Comment by gogogadgetbioprocess** [comment]
So What shall we do about this?

**Comment by chr1sgrant** [comment]
I guess this can be deleted now

**Pull request closed without merging**